mevolib/inference/__init__.py

```diff
@@ -32,14 +45,18 @@
 import tempfile
 import subprocess
 
-from Bio import AlignIO, Phylo
+import Bio.Phylo.BaseTree
+
+from typing import Optional
+
+from io import StringIO
+from Bio import AlignIO,Phylo
+
 
 from . import _FastTree
```

Comment:
@JAlvarezJarreta : I think this suggestion should work. Do you have any preferences about this?

```suggestion
from mevolib.inference import _FastTree, _RAxML
```